documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+  - Until breaking change detection tools are available, the feature teams and SDK team will use code reviews to prevent breaking changes
+- In the event that a breaking change is released outside of a planned breaking change release, the cmdlet owner must coordinate with the SDK team to create a hotfix that mitigates the impact of the breaking changes on existing scripts.
+- The release notes for any breaking change version will contain a complete description of the changes and a migration guide, explaining how scripts should be updated to accomodate the breaking changes.
+  - Service Teams must provide a list of breaking changes and the migration guide at the start of the sprint containing the breaking changes
+  - The SDK Team will coordinate the breaking change documentation and assemble a migration guide from the feature team submissions
+  - A sample migration guide for the 2.0 release is provided [here](https://github.com/Azure/azure-powershell/blob/dev/documentation/release-notes/migration-guide.2.0.0.md)
+  - The breaking change list and migration guide will be distributed to customers in the github release, in an azure.com blog post, and in powershell help topics
+
+### Exceptions
+The majority of cmdlet enhancement and fixes should not require breaking changes.  However, to accomodate extraordinary circumstances, exceptions to the breaking change policy can be made for the following reasons:
+- High impact security bugs, data loss, or other severe functionality issues that can only be resolved through a breaking change
+- Important fixes and new features that require a breaking change, for which usage data or extensive customer outreach shows extremely low impact on customer scripts
+
+##  Preview Modules
+
+The breaking change policy should not prevent providing cmdlet support for non-GA services or experimenting with new cmdlets for existing services.  To satisfy these two needs, Azure PowerShell will begin supporting Preview modules in the October release.
+
+Note that, participation in Preview modules is entirely volunatry for feature teams.  If cmdlets for an Azure service remain stable, the service team may decide not to support a preview module.
+
+- Preview modules use the suffix Preview in their module name, as in 'AzureRM.Compute.Preview' 
+- Preview modules will not be included in the AzureRM module in the PowerShell Gallery, or in the released PowerShell MSI.
+- A new roll-up module, AzureRM.Preview will reference the existing set of preview modules, allowing uses to download all of them at once.
+- Non-GA services will automatically begin as Preview modules. However, a full set of non-preview cmdlets must be available in the PowerShell Gallery and PowerShell msi within 3 months of service GA.
+- GA Services may *also* have a preview module to use when experimenting with new cmdlets
+- Preview modules will *not* use semantic versioning
+
+## Breaking Change Definition
+
+Breaking changes in cmdlets are defined as follows:
```

Comment:
@brendanburns  We are specifically limiting this to breaking the API surface for powershell scripts.  We have left the behavior breaks out of the equation, because they are difficult to quantify.